oz/sqlalchemy/__init__.py

```diff
@@ -6,16 +6,15 @@
 from sqlalchemy.orm.session import sessionmaker
 from sqlalchemy.ext.declarative import declarative_base
 from multiprocessing.util import register_after_fork
-import tornado.options
 import oz
 
 from .actions import *
 from .middleware import *
 from .options import *
 
 Base = declarative_base()
-_engine = None
-_session = None
+_engines = {}
```

Comment:
Think we could use `dict()` here to make it clear we are using a dictionary constructor and not an empty set constructor?